### ISSUE TITLE
Scrolling too slow

### ISSUE THREAD

**Issue by mocoelho** [horosproject/horos#538]
why is scrolling images too slow? It looks like a ruge lag, just tested the same exam in Osirix and scrolling is normal.
It seems to be more slow in multiphase CT studies

**Comment by JeanClaude2** [comment]
Yes, this is an established fact : OsiriX is much better than Horos.

It seems that since Horos was forked from OsiriX, it has become less and less usable. Purview, the company that's driving its development, seems more interested in selling its products than actually fixing/developing Horos.

Let's hope that their Feb. 14th announcement will concerned the dev of Horos and not a new $$$ product.

**Comment by sorenchr2011** [comment]
I am glad there is an open source and free alternative to Osirix.
It's possible for everyone to improve Horos if they wish to speed up
development. Would you rather have no one spending time on it at all?

Soren

**Comment by raybd83** [comment]
Same issue here. For me this is the only major issue to be fixed. Others issues are minor.
This slow scroll seems to happen just with series with a lot of images, mainly  CT. Today it is very difficult to use horos to see large series images.  Osirix and OnePacs do well in this situations.  As of today, an alternative to see this large series with Horous is using 2D Orthogonal View (very fast scrolling in this mode).

**Comment by raybd83** [comment]
Same issue here. For me this is the only major issue to be fixed. Others issues are minor.
This slow scroll seems to happen just with series with a lot of images, mainly  CT. Today it is very difficult to use horos to see large series images.  Osirix and OnePacs do well in this situations.  As of today, an alternative to see this large series with Horous is using 2D Orthogonal View (very fast scrolling in this mode).

**Comment by ferrazza** [comment]
Hi,
I have the same issue on macOS Catalina with all tried version above the 2.2.0, even 4.0.0. The last version in which scrolling works smoothly is the 2.2.0. This is a major issue and should be solved because Horos is useless with it!
